buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -797,6 +798,7 @@ class BeamModulePlugin implements Plugin<Project> {
         hamcrest                                    : "org.hamcrest:hamcrest:$hamcrest_version",
         http_client                                 : "org.apache.httpcomponents:httpclient:$httpclient_version",
         http_core                                   : "org.apache.httpcomponents:httpcore:$httpcore_version",
+        iceberg_bqms_catalog                        : "https://storage.googleapis.com/spark-lib/bigquery/iceberg-bigquery-catalog-${iceberg_bqms_catalog_version}.jar",
```

Comment:
Can we move this logic to the specific module and just keep the constant here ? At least let's add a TODO here to update when the jar is released to a Maven repository.

Reply:
By "constant" do you mean the GCS path, or the supposed version?

Reply:
At this point, does it make sense to just put everything inside the module?

Reply:
I meant to just define the version number here but move the library path definition to the module. But if this is not possible let's just add the TODO.

Reply:
done, PTAL.